app_dev.yaml

```diff
@@ -17,7 +17,17 @@ inbound_services:
 
 default_expiration: "10m"
 handlers:
-# PRODUCTION STATIC
+# Note: The handlers in this file contain two separate sections:
```

Comment:
Will non-overridden handlers in the DEPLOYMENT section be available in local setup? If so, you should state that in the documentation for this file.

Also state which section is applicable for runs on Travis or CircleCI.

I worry a bit about this setup, btw. It introduces skew between deployment and non-deployment. The end result is that we may only discover that there are bugs in app.yaml when we actually deploy to a test server, and these won't be caught in the Travis tests. Do you have any thoughts on how to mitigate/minimize that (e.g. making the number of DEPLOYMENT handlers as small as possible)?